### DISCUSSION TITLE
Datasafe code coverage

### DISCUSSION THREAD

**Discussion by francis-pouatcha** [adorsys/datasafe#273] [General]
We want to use this discussion to workout details on the implementation of the code coverage inside datasafe.

We have noticed that datasafe has following lines, and we would like to have them explained in detail by the current maintenance team.

**Comment by NdSaid** [comment]
There are the lines 
[![Build Status](https://travis-ci.com/adorsys/datasafe.svg?branch=develop)](https://travis-ci.com/adorsys/datasafe)
[![codecov](https://codecov.io/gh/adorsys/datasafe/branch/develop/graph/badge.svg)](https://codecov.io/gh/adorsys/datasafe)
[![Maintainability](https://api.codeclimate.com/v1/badges/06ae7d4cafc3012cee85/maintainability)](https://codeclimate.com/github/adorsys/datasafe/maintainability)
